bazaarvoice/emodb
# new postman tests for put /uac/1/role/{group}/{id}

## Github Issue #
## What Are We Doing Here?

We have created put /uac/1/role/group/id endpoint tests with postman
## How to Test and Verify

1. Check out this PR
2. Run this branch on CI with pipeline stage for running these tests 
3. Profit

## Risk
new broken/unstable tests have been added to test suit

### Level 

`Low`

### Required Testing

`Manual`
### Risk Summary

This doesn't affect existing functionality but it can break stability of existing test suite

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
